### ISSUE TITLE
Settings "Exclude Libraries" does not support spaces

### ISSUE THREAD

**Issue by JHellwig** [damies13/rfswarm#171]
**Describe the bug**
Settings "Exclude Libraries" does not support spaces, if you separate with comma and a space, the test will throw errors. It only works when there are no spaces.

**To Reproduce**
Use for Exclude Libraries the String: "String, SeleniumLibrary"

**Expected behavior**
Should exclude both libraries.

**Comment by damies13** [comment]
Thanks Jascha for the bug report.

**Comment by damies13** [comment]
When the default exclude libraries are modified as `BuiltIn,String, OperatingSystem,perftest`

The robot command line stdout returns:
```
[ ERROR ] Parsing 'OperatingSystem,perftest' failed: File or directory to execute does not exist.

Try --help for usage information.
```

This confirms what the issue is, it's a bit different from what I expected, but now I know I can reproduce it.

**Comment by damies13** [comment]
Note for later:
https://stackoverflow.com/questions/12182777/is-there-a-better-way-to-use-strip-on-a-list-of-strings-python

**Comment by damies13** [comment]
All test passed, 3 failed tests were for Python 3.12 and for Issue #177 

The fix for this issue has been merged into release branch v1.2.1
